zanata-war/src/main/java/org/zanata/service/impl/LocaleServiceImpl.java

```diff
@@ -89,17 +90,18 @@ public LocaleServiceImpl(LocaleDAO localeDAO, ProjectDAO projectDAO,
     }
 
     public static Map<LocaleId, String> getLocaleAliasesByIteration(HProjectIteration iteration) {
-        Map<LocaleId, String> localeAliases = null;
+        Map<LocaleId, String> localeAliases;
```

Comment:
You don't need to declare this map at all. Just put the return statement in line 96 and 98.